worldedit-core/src/main/java/com/fastasyncworldedit/core/extent/DisallowedBlocksExtent.java

```diff
@@ -134,34 +133,67 @@ public IChunkSet processSet(final IChunk chunk, final IChunkGet get, final IChun
             if (!set.hasSection(layer)) {
                 continue;
             }
-            char[] blocks = Objects.requireNonNull(set.loadIfPresent(layer));
-            it:
-            for (int i = 0; i < blocks.length; i++) {
-                char block = blocks[i];
-                BlockState state = states[block];
-                if (blockedBlocks != null) {
-                    if (blockedBlocks.contains(state.getBlockType().getId())) {
-                        blocks[i] = 0;
+
+            if (!(set instanceof IntSetBlocks)) {
```

Comment:
Still a couple places with the negated if checks

Reply:
Fixed in latest commit! Thanks for catching these.